templates/status.php

```diff
@@ -60,6 +60,13 @@
     echo(present_attributes($this, $list, ''));
 }
 
+$authData = $this->data['authData'];
+if (isset($authData)) {
```

Comment:
This will always be set because it's set right above. Maybe intended was `!empty()`?